update-license/main.go

```diff
@@ -80,19 +72,61 @@ func main() {
 
 func do() error {
 	flag.Parse()
+
 	if len(flag.Args()) < 1 {
 		return fmt.Errorf("usage: %s GO_FILES", os.Args[0])
 	}
 
-	return updateFiles(flag.Args(), time.Now().UTC().Year(), *flagDryRun)
+	return updateFiles(
+		flag.Args(),
+		time.Now().UTC().Year(),
+		*flagLicense,
+		*flagOwner,
+		*flagDryRun,
+	)
 }
 
-func updateFiles(filePaths []string, year int, dryRun bool) error {
+func fullLicense(ts string, year int, owner string) string {
+	var buf bytes.Buffer
+	t, err := template.New("").Parse(ts)
+	if err != nil {
+		log.Fatal("failed to parse license remplate", err)
+	}
+
+	data := struct {
+		Year  int
+		Owner string
+	}{year, owner}
+	if err := t.Execute(&buf, data); err != nil {
+		log.Fatal("failed to execture license template", err)
+	}
+
+	return strings.TrimSpace(buf.String())
+}
+
+// validLicenses grabs all the license templates from the folder
+func validLicenses() []string {
+	res := []string{}
+
+	for k, _ := range licenseTemplates {
+		res = append(res, k)
+	}
+
+	return res
```

Comment:
Some determinism here would be preferable. Otherwise `--help` will keep
changing the order of the list of valid licenses.

```suggestion
	sort.Strings(res)
	return res
```